tests/conftest.py

```diff
@@ -52,6 +54,47 @@ def loader(relpath, **kwargs):
     return loader
 
 
+def checkCarlaPath():
+    CARLA_ROOT = os.environ.get("CARLA_ROOT")
+    if not CARLA_ROOT:
+        pytest.skip("CARLA_ROOT env variable not set.")
+    return CARLA_ROOT
+
+
+@pytest.fixture
+def launchCarlaServer():
```

Comment:
Let's move all these CARLA-specific things into `tests/simulators/carla/conftest.py`.

Reply:
Ack! Will do that accordingly!